.github/workflows/8jre-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and push the Docker image for JRE 8
+
+on:
+  push:
+    tags:
+      - "8jre-[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/8jre##g")
+          echo ::set-output name=version::${VERSION}
+
+      - name: Check out codes
+        uses: actions/checkout@v2
+
+      - name: Build Docker image
+        run: docker build -t ghcr.io/scalar-labs/8jre:${{ steps.version.outputs.version }} ./jre/8
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: ghcr.io/scalar-labs/8jre:${{ steps.version.outputs.version }}
```

Comment:
```suggestion
          image-ref: ghcr.io/scalar-labs/jre8:${{ steps.version.outputs.version }}
```
I think `<runtime><version>` is easier to manage but is there any particular reason for the naming?
(it is also inconsistent with the directory structure (jre/8))

Reply:
@feeblefakie 
Understood.
Let me revise the naming.
(There's no particular reason for 8jre. Sorry for my bad naming... )

Reply:
@feeblefakie 
I changed the naming in https://github.com/scalar-labs/docker/pull/1/commits/7be024e51df7aa6060f9e598db9bd0c35cd82db3
Please take a look when you have a chance.
Thank you.

If it's possible, could you create a new secret `CR_PAT` of your personal access token?
Thank you :D

Reply:
I think CR_PAT is configured in organization-wide, so should be fine.